hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -3098,13 +3097,18 @@ static void createRecordIndexDefinition(HoodieTableMetaClient metaClient,
    * a very large number of files are present in the dataset being initialized.
    */
   public static class DirectoryInfo implements Serializable {
+
     // Relative path of the directory (relative to the base directory)
+    @Getter(AccessLevel.PACKAGE)
```

Comment:
nit: `@Getter(AccessLevel.PACKAGE)` at the class level

Reply:
Done